app/commands/git/sync_concept_exercise.rb

```diff
@@ -18,8 +18,7 @@ def call
         status: exercise_config[:status] || :active,
         icon_name: head_git_exercise.icon_name,
         position: exercise_position,
-        # TODO: Remove the || ... once we have configlet checking things properly.
-        title: exercise_config[:name].presence || exercise_config[:slug].titleize,
+        title: exercise_config[:name].presence,
```

Comment:
Configlet is checking this field, so it can be removed as then tracks will be notified when this fails.